datahub-graphql-core/src/main/resources/entity.graphql

```diff
@@ -11054,6 +11054,10 @@ input CreateDataProductInput {
     The primary key of the Domain
     """
     domainUrn: String!
+    """
+    An id for the DataProduct
```

Comment:
nitpick: let's say 

"An optional id for the new data product"

Reply:
Done